Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ ly_add_target(
             Gem::PhysX.Static
 )
 
-target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs urdfdom tf2_ros ackermann_msgs gazebo_msgs control_toolbox)
+target_depends_on_ros2_packages(${gem_name}.Static rclcpp rclcpp_action builtin_interfaces std_msgs sensor_msgs nav_msgs control_msgs urdfdom tf2_ros ackermann_msgs gazebo_msgs control_toolbox)
```

Comment:
This `rclcpp_action` need a new ros package installation requirement? If so, the README.md needs to be updated